trunk/src/app/srs_app_rtc_conn.cpp

```diff
@@ -3171,6 +3175,11 @@ srs_error_t SrsRtcConnection::generate_publish_local_sdp(SrsRequest* req, SrsSdp
         }
     }
 
+    if (local_sdp.media_descs_.empty()){
+        srs_trace("empty media desc!");
```

Comment:
Please remove this line, because all information is in err, no need to logging here.